200-intranetstyling.patch

* Hides checkin/renew/search-patrons from search headers
* Make patron filters autofocus on patron page
* Add focus to search filter on patrons page, set search fields to date of birth

diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/adv-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/adv-search.inc
index 4bce9c4..fbee3b0 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/adv-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/adv-search.inc
@@ -37,13 +37,13 @@
                 <li><a class="keep_text" href="#circ_search">Check out</a></li>
             [% END %]
             [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]
-                <li><a class="keep_text" href="#checkin_search">Check in</a></li>
+                <li class="deichman-hidden"><a class="keep_text" href="#checkin_search">Check in</a></li>
             [% END %]
             [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]
-                <li><a class="keep_text" href="#renew_search">Renew</a></li>
+                <li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>
             [% END %]
             [% IF ( CAN_user_borrowers ) %]
-                <li><a class="keep_text" href="#patron_search">Search patrons</a></li>
+                <li class="deichman-hidden"><a class="keep_text" href="#patron_search">Search patrons</a></li>
             [% END %]
         </ul>

diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/cat-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/cat-search.inc
index 972eb0c..8b18b17 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/cat-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/cat-search.inc
@@ -35,8 +35,8 @@

 <ul>
     [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#circ_search">Check out</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
     [% IF ( CAN_user_catalogue ) %]<li class="keep_text ui-tabs-active"><a href="#catalog_search">Search the catalog</a></li>[% END %]
 </ul>

diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/cataloging-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/cataloging-search.inc
index a289b70..d289696 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/cataloging-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/cataloging-search.inc
@@ -42,8 +42,8 @@
         <ul>
             <li><a class="keep_text" href="#addbooks_search">Cataloging search</a></li>
             [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#circ_search">Check out</a></li>[% END %]
-            [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
-            [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
+            [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
+            [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
             <li><a class="keep_text" href="#catalog_search">Search the catalog</a></li>
         </ul>
     </div><!-- /header_search -->
diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/checkin-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/checkin-search.inc
index edaacee..9231c7f 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/checkin-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/checkin-search.inc
@@ -25,7 +25,7 @@

 <ul>
     [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#circ_search">Check out</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
     [% IF ( CAN_user_catalogue ) %]<li><a class="keep_text" href="#catalog_search">Search the catalog</a></li>[% END %]
 </ul>

diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/circ-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/circ-search.inc
index 7fec2dd..c2bdf12 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/circ-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/circ-search.inc
@@ -33,8 +33,8 @@

 <ul>
     [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#circ_search">Check out</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
     [% IF ( CAN_user_catalogue ) %]<li><a class="keep_text" href="#catalog_search">Search the catalog</a></li>[% END %]
 </ul>

diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/home-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/home-search.inc
index be87ac2..e7ecade 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/home-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/home-search.inc
@@ -42,9 +42,9 @@

 <ul>
     [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#circ_search">Check out</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
-    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
-    [% IF ( CAN_user_borrowers ) %]<li><a class="keep_text" href="#patron_search">Search patrons</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
+    [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
+    [% IF ( CAN_user_borrowers ) %]<li class="deichman-hidden"><a class="keep_text" href="#patron_search">Search patrons</a></li>[% END %]
     [% IF ( CAN_user_catalogue ) %]<li><a class="keep_text" href="#catalog_search">Search the catalog</a></li>[% END %]
 </ul>

diff --git a/koha-tmpl/intranet-tmpl/prog/en/includes/patron-search.inc b/koha-tmpl/intranet-tmpl/prog/en/includes/patron-search.inc
index a7e11b3..dc832f2 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/includes/patron-search.inc
+++ b/koha-tmpl/intranet-tmpl/prog/en/includes/patron-search.inc
@@ -147,10 +147,10 @@
     </div>[% END %]

     <ul>
-        <li><a class="keep_text" href="#patron_search">Search patrons</a></li>
+        <li class="deichman-hidden"><a class="keep_text" href="#patron_search">Search patrons</a></li>
         [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#circ_search">Check out</a></li>[% END %]
-        [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
-        [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
+        [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#checkin_search">Check in</a></li>[% END %]
+        [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]<li class="deichman-hidden"><a class="keep_text" href="#renew_search">Renew</a></li>[% END %]
         [% IF ( CAN_user_catalogue ) %]<li><a class="keep_text" href="#catalog_search">Search the catalog</a></li>[% END %]
     </ul>
 </div><!-- /header_search -->
diff --git a/koha-tmpl/intranet-tmpl/prog/en/modules/intranet-main.tt b/koha-tmpl/intranet-tmpl/prog/en/modules/intranet-main.tt
index b251d9b..ff90995 100644
--- a/koha-tmpl/intranet-tmpl/prog/en/modules/intranet-main.tt
+++ b/koha-tmpl/intranet-tmpl/prog/en/modules/intranet-main.tt
@@ -39,6 +39,9 @@ var MSG_CONFIRM_DELETE = _("Are you sure you want to delete this news item? This
                 <ul class="biglinks-list">
                     [% IF ( CAN_user_circulate_circulate_remaining_permissions ) %]
                     <li>
+                       <a class="icon_general icon_circulation" href="/cgi-bin/koha/circ/returns.pl">Lever inn</a>
+                    </li>
+                    <li>
                         <a class="icon_general icon_circulation" href="/cgi-bin/koha/circ/circulation-home.pl">Circulation</a>
                     </li>
                     [% END %]
